braindecode/models/tcn.py

```diff
@@ -50,7 +48,6 @@ def __init__(
             n_filters=None,
             kernel_size=None,
             drop_prob=None,
-            add_log_softmax=None,
```

Comment:
this parameter should stay here but with default value set to True (to maintain the behavior implemented now in this PR). It should be also back in the docstring with small adjustement. Value of `add_log_softmax` should be used in `super().__init__`